make-a-pancake.md

```diff
@@ -0,0 +1,2 @@
+If you want you can make a pancake with your special marshmallow cat. You can make nutella crepes, or big fluffy pancakes with maple syrup.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.